scripts/build.sh

```diff
@@ -546,7 +546,7 @@ for (( i=0 ; i < ${#BUILDS[@]} ; i++ )) ; do
 	else
 		EXTRA_OPTS="-DNO_USERMANUAL=ON -DNO_PRINTING=ON"
 	fi
-	if [ "$SUBSURFACE_EXECUTABLE" = "DesktopExecutable" ] && [ "$BUILD_WITH_WEBENGINE" = "1" ]); then
+	if [ "$SUBSURFACE_EXECUTABLE" = "DesktopExecutable" ] && [ "$BUILD_WITH_WEBENGINE" = "1" ]; then
```

Comment:
I hate it when I do stupid things like this.
I tested it, then did a "quick edit" and committed crud like this. Thanks for the fix... I'll rebase this of course